html_css/v2/main-course/selectors.md

```diff
@@ -0,0 +1,264 @@
+### Introduction
+By now you should be comfortable with basic CSS selectors and have no trouble grabbing things by their type, class or ID. But to be a real CSS surgeon, sometimes you need more specialized tools. In this lesson we'll look at advanced CSS selectors and show you how to target elements in a more specific and finely grained way.
+
+These selectors can be especially useful when you can't (or don't want to) change your HTML markup.
+
+There are _a lot_ of advanced selectors, so going through every single one is outside the scope of this lesson. However, we'll go through some of the most useful and common ones, as well as arm you with the concepts and vocabulary to learn more on your own.
+
+As always feel free to open up your code editor and run you own experiments with these selectors - practice makes perfect!
+
+### Learning Outcomes
+* Understand how to use parent and sibling selectors
+* Recognize the difference between pseudo classes and pseudo elements
+* Learn about some of the most useful and common pseudo elements and pseudo classes
+* Learn about the different ways to select an attribute or its parts
+
+### Parent and Sibling Combinators
+Let's have a look at some more ways we can access different elements _without_ referring to their classes. Here are three new selectors to do just that.
+
+* `>` - the child combinator
+* `+` - the adjacent sibling combinator
+* `~` - the general sibling combinator
+
+We'll tackle some practical examples using this sample markup.
+
+~~~html
+<main class="parent">
+  <div class="child group1">
+    <div class="grand-child group1"></div>
+  </div>
+  <div class="child group2">
+    <div class="grand-child group2"></div>
+  </div>
+  <div class="child group3">
+    <div class="grand-child group3"></div>
+  </div>
+</main>
+~~~
+
+By now, you should be pretty comfortable writing rules using the descendant combinator you learned about in [foundations](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/css-foundations). For instance, if we wanted to select all the `child` and `grand-child` divs inside of `main`, we could write:
+
+~~~css
+main div {
+  /* Our cool CSS */
+}
+~~~
+
+But what if we wanted to be more specific and select _only_ the `child` or `grand-child` divs ? That's where the child combinator `>` comes in handy. Unlike the descendant combinator, it will only select direct children.
+
+~~~css
+/* This rule will only select divs with a class of child */
+main > div {
+  /* Our cool CSS */
+}
+
+/* This rule will only select divs with a class of grand-child */
+main > div > div {
+  /* More cool CSS */
+}
+~~~
+
+Phrased another way, the child selector will select an element that is one level of indentation down. In order to select an element that is adjacent to our target, or on the same level of indentation, we can use the adjacent sibling combinator `+`.
+
+~~~css
+/* This rule will only select the div with the classes child group2 */
```

Comment:
Might sound better and be more descriptive (selecting class, not classes) if worded as. 

/* This rule will only select the div with the class of child group2 */